code/script.js

```diff
@@ -0,0 +1,98 @@
+// The strict mode in JavaScript prevents several bugs. For example, it won't let you use undeclared variables. 
+"use strict";
+
+// Global variables
+const USER = 'isomoth';
+const USER_URL = `https://api.github.com/users/${USER}`;
+const REPOS_URL = `https://api.github.com/users/${USER}/repos`;
+const userInfo = document.getElementById('user-info');
+const projectsContainer = document.getElementById('project-grid');
+
+// Fetch user info and inject it into a div
+const getUserProfile = () => {
+  fetch(USER_URL)
+    .then((response) => response.json())
+    .then((data) => {
+      userInfo.innerHTML += `
+      <div class="user-info">
+        <img class="user-image" src='${data.avatar_url}'/>
+        <div class="user-names">
+          <h2 class="nickname">${data.name}</h2>
+          <h3><a href="https://github.com/isomoth">${data.login}</a></h3>
+          <p class="bio">${data.bio}</p>
```

Comment:
Really nice that you also hade this bio on your site!

Reply:
also... You make the code look so simple! I really enjoy reading through it, i can for sure take some inspiration from your code!

Reply:
Thanks a lot for your kind words Hedvig! It's nice to have a different perspective on the solutions I implemented. Much appreciated!